python/py-kaptan/Portfile

```
@@ -13,7 +13,8 @@ license             BSD
 maintainers         {@egorenar posteo.net:egorenar-dev} openmaintainer
 
 description         Configuration parser.
-long_description    ${description}
+long_description    {*}Configuration parser with dict, JSON, YAML, .ini \
```

```suggestion
long_description    Configuration parser with dict, JSON, YAML, .ini \
```

Sorry if I wasn't clear on this.`{*}` isn't necessary for strings, it's simply used for [expansion](https://wiki.tcl-lang.org/page/%7B%2A%7D). It would however have been necessary if it was `{*}${description}`. 

Thanks for updating the long description.